### DISCUSSION TITLE
Custom Function Development for Luma AI-Level Quality

### DISCUSSION THREAD

**Discussion by Vision70s** [mkkellogg/GaussianSplats3D#391] [Q&A]
Hello everyone!

I’m currently using this code in my projects and I’m exploring the possibility of developing custom functions based on it for our business solutions. I wanted to ask:
	1.	Are there people in the community who would be interested in working on customizing this code for payment?
	2.	What would be the approximate cost of such services?

If you have experience offering similar services or know how this could be organized, I’d greatly appreciate any recommendations or proposals. Feel free to share your insights or reach out to me directly for further discussion.

Thank you in advance for your responses!

**Comment by ramaramx** [comment]
what you expectation @Vision70s ?
can we personal chat ?
feel free chat rambudiarto@gmail.com

**Comment by koktavy** [comment]
@Vision70s Hi, I'm an immersive web engineer open to freelance, feel free to reach out! tavius.koktavy@gmail.com
